fiftyone/utils/cvat.py

```diff
@@ -1231,32 +1108,22 @@ def __init__(self, occluded=None, attributes=None):
         self.attributes = attributes or []
 
     def _to_attributes(self):
-        attributes = {a.name: a.to_attribute() for a in self.attributes}
+        attributes = {a.name: a.value for a in self.attributes}
 
         if self.occluded is not None:
-            attributes["occluded"] = fol.BooleanAttribute(value=self.occluded)
+            attributes["occluded"] = self.occluded
```

Comment:
Hmmm, this errors on occluded boolean attributes. @ehofesmann is this expected?

```py
import fiftyone as fo

dataset = fo.Dataset()
sample = fo.Sample("path/to/image.png")
sample["detection"] = fo.Detection(occluded=True, bounding_box=[0, 0, 1, 1]) 
dataset.add_sample(sample)

dataset.export(
    export_dir="/tmp/cvat-test-export", dataset_type=fo.types.CVATImageDataset
)
```

Reply:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-5be5aac31b16> in <module>
      6 dataset.add_sample(sample)
      7 
----> 8 dataset.export(
      9     export_dir="~/tmp/cvat-test-export", dataset_type=fo.types.CVATImageDataset
     10 )

~/code/fiftyone/fiftyone/core/collections.py in export(self, export_dir, dataset_type, data_path, labels_path, export_media, dataset_exporter, label_field, label_prefix, labels_dict, frame_labels_field, frame_labels_prefix, frame_labels_dict, overwrite, **kwargs)
   5557 
   5558         # Perform the export
-> 5559         foud.export_samples(
   5560             self,
   5561             dataset_exporter=dataset_exporter,

~/code/fiftyone/fiftyone/utils/data/exporters.py in export_samples(samples, export_dir, dataset_type, data_path, labels_path, export_media, dataset_exporter, label_field_or_dict, frame_labels_field_or_dict, num_samples, **kwargs)
    289         )
    290 
--> 291     write_dataset(
    292         samples,
    293         sample_parser,

~/code/fiftyone/fiftyone/utils/data/exporters.py in write_dataset(samples, sample_parser, dataset_exporter, num_samples, sample_collection)
    342         (UnlabeledImageDatasetExporter, LabeledImageDatasetExporter),
    343     ):
--> 344         _write_image_dataset(
    345             dataset_exporter,
    346             samples,

~/code/fiftyone/fiftyone/utils/data/exporters.py in _write_image_dataset(dataset_exporter, samples, sample_parser, num_samples, sample_collection)
    670                 else:
    671                     # Export sample
--> 672                     dataset_exporter.export_sample(
    673                         image_or_path, metadata=metadata
    674                     )

~/code/fiftyone/fiftyone/utils/data/exporters.py in __exit__(self, *args)
   1006 
   1007     def __exit__(self, *args):
-> 1008         self.close(*args)
   1009 
   1010     def setup(self):

~/code/fiftyone/fiftyone/utils/cvat.py in close(self, *args)
    489         if self._task_labels is None:
    490             # Compute task labels from active label schema
--> 491             cvat_task_labels = CVATTaskLabels.from_cvat_images(
    492                 self._cvat_images
    493             )

~/code/fiftyone/fiftyone/utils/cvat.py in from_cvat_images(cls, cvat_images)
    741 
    742                 if anno.occluded is not None:
--> 743                     schema.add_object_attribute("occluded", anno.occluded)
    744 
    745                 for attr in anno.attributes:

~/code/fiftyone/eta/eta/core/frames.py in add_object_attribute(self, label, attr)
   1395             attr: an Attribute
   1396         """
-> 1397         self.objects.add_object_attribute(label, attr)
   1398 
   1399     def add_object_attributes(self, label, attrs):

~/code/fiftyone/eta/eta/core/objects.py in add_object_attribute(self, label, attr)
   1726         """
   1727         self._ensure_has_object_label(label)
-> 1728         self.schema[label].add_object_attribute(attr)
   1729 
   1730     def add_object_attributes(self, label, attrs):

~/code/fiftyone/eta/eta/core/objects.py in add_object_attribute(self, attr)
   1234             attr: an `eta.core.data.Attribute`
   1235         """
-> 1236         self.attrs.add_attribute(attr)
   1237 
   1238     def add_object_attributes(self, attrs):

~/code/fiftyone/eta/eta/core/data.py in add_attribute(self, attr)
   1244             attr: an Attribute
   1245         """
-> 1246         name = attr.name
   1247         if name not in self.schema:
   1248             schema_cls = attr.get_schema_cls()

AttributeError: 'bool' object has no attribute 'name'
```

Reply:
I'll push a fix for this. Double checking that there are no other issues